style/header.less

```diff
@@ -1,15 +1,26 @@
 @import './variables.less';
 
-.rdg-header {
-  background: #f9f9f9;
-  overflow: hidden;
+.rdg-header-row,
+.rdg-filter-row {
+  width: var(--row-width);
+  position: -webkit-sticky;
+  position: sticky;
+  background-color: #f9f9f9;
   font-weight: bold;
+  .user-select();
+  z-index: 3;
 }
 
 .rdg-header-row {
-  position: relative;
-  background-color: inherit;
-  .user-select();
+  height: var(--header-row-height);
+  line-height: var(--header-row-height);
+  top: 0;
+}
+
+.rdg-filter-row {
+  height: var(--filter-row-height);
```

Comment:
Can I ask how can we navigate the definition of the css variable? For example, click on the variable, and hit F12 in vs code, whether it will jump to the `tsx` code?

Reply:
There's no navigation for this, aside from the trusty <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>f</kbd>.